Sender: LSF System <lsfadmin@eu-g3-025>
Subject: Job 217208791: <classification_typicaldec_conservative_50shot> in cluster <euler> Exited

Job <classification_typicaldec_conservative_50shot> was submitted from host <eu-login-17> by user <jmattern> in cluster <euler> at Wed May  4 15:57:07 2022
Job was executed on host(s) <20*eu-g3-025>, in queue <gpu.24h>, as user <jmattern> in cluster <euler> at Wed May  4 15:57:14 2022
</cluster/home/jmattern> was used as the home directory.
</cluster/work/sachan/jmattern/private-datasets> was used as the working directory.
Started at Wed May  4 15:57:14 2022
Terminated at Wed May  4 15:57:16 2022
Results reported at Wed May  4 15:57:16 2022

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
python train_classifier.py --gen_data --device 0 --pos-file positive_generated_typical_conservative.txt --neg-file negative_generated_typical_conservative.txt
------------------------------------------------------------

Exited with exit code 127.

Resource usage summary:

    CPU time :                                   0.21 sec.
    Max Memory :                                 15 MB
    Average Memory :                             -
    Total Requested Memory :                     20480.00 MB
    Delta Memory :                               20465.00 MB
    Max Swap :                                   -
    Max Processes :                              -
    Max Threads :                                -
    Run time :                                   2 sec.
    Turnaround time :                            9 sec.

The output (if any) follows:

python: error while loading shared libraries: libpython3.8.so.1.0: cannot open shared object file: No such file or directory
Sender: LSF System <lsfadmin@eu-g3-025>
Subject: Job 217209093: <classification_typicaldec_conservative_50shot> in cluster <euler> Exited

Job <classification_typicaldec_conservative_50shot> was submitted from host <eu-login-17> by user <jmattern> in cluster <euler> at Wed May  4 15:58:03 2022
Job was executed on host(s) <20*eu-g3-025>, in queue <gpu.24h>, as user <jmattern> in cluster <euler> at Wed May  4 15:58:15 2022
</cluster/home/jmattern> was used as the home directory.
</cluster/work/sachan/jmattern/private-datasets> was used as the working directory.
Started at Wed May  4 15:58:15 2022
Terminated at Wed May  4 15:58:20 2022
Results reported at Wed May  4 15:58:20 2022

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
python train_classifier.py --gen_data --device 0 --pos-file positive_generated_typical_conservative.txt --neg-file negative_generated_typical_conservative.txt
------------------------------------------------------------

Exited with exit code 1.

Resource usage summary:

    CPU time :                                   0.88 sec.
    Max Memory :                                 67 MB
    Average Memory :                             -
    Total Requested Memory :                     20480.00 MB
    Delta Memory :                               20413.00 MB
    Max Swap :                                   -
    Max Processes :                              -
    Max Threads :                                -
    Run time :                                   4 sec.
    Turnaround time :                            17 sec.

The output (if any) follows:

Traceback (most recent call last):
  File "train_classifier.py", line 4, in <module>
    import torch.optim as optim
  File "/cluster/home/jmattern/.local/lib/python3.8/site-packages/torch/optim/__init__.py", line 8, in <module>
    from .adadelta import Adadelta
  File "/cluster/home/jmattern/.local/lib/python3.8/site-packages/torch/optim/adadelta.py", line 3, in <module>
    from . import _functional as F
  File "/cluster/home/jmattern/.local/lib/python3.8/site-packages/torch/optim/_functional.py", line 4, in <module>
    from torch import Tensor
ImportError: cannot import name 'Tensor' from 'torch' (unknown location)
Sender: LSF System <lsfadmin@eu-g3-007>
Subject: Job 217210159: <classification_typicaldec_conservative_50shot> in cluster <euler> Exited

Job <classification_typicaldec_conservative_50shot> was submitted from host <eu-login-17> by user <jmattern> in cluster <euler> at Wed May  4 16:01:03 2022
Job was executed on host(s) <20*eu-g3-007>, in queue <gpu.24h>, as user <jmattern> in cluster <euler> at Wed May  4 16:01:16 2022
</cluster/home/jmattern> was used as the home directory.
</cluster/work/sachan/jmattern/private-datasets> was used as the working directory.
Started at Wed May  4 16:01:16 2022
Terminated at Wed May  4 16:01:19 2022
Results reported at Wed May  4 16:01:19 2022

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
python train_classifier.py --gen_data --device 0 --pos-file positive_generated_typical_conservative.txt --neg-file negative_generated_typical_conservative.txt
------------------------------------------------------------

Exited with exit code 1.

Resource usage summary:

    CPU time :                                   0.83 sec.
    Max Memory :                                 30 MB
    Average Memory :                             -
    Total Requested Memory :                     20480.00 MB
    Delta Memory :                               20450.00 MB
    Max Swap :                                   -
    Max Processes :                              -
    Max Threads :                                -
    Run time :                                   3 sec.
    Turnaround time :                            16 sec.

The output (if any) follows:

Traceback (most recent call last):
  File "train_classifier.py", line 4, in <module>
    import torch.optim as optim
  File "/cluster/home/jmattern/.local/lib/python3.8/site-packages/torch/optim/__init__.py", line 8, in <module>
    from .adadelta import Adadelta
  File "/cluster/home/jmattern/.local/lib/python3.8/site-packages/torch/optim/adadelta.py", line 3, in <module>
    from . import _functional as F
  File "/cluster/home/jmattern/.local/lib/python3.8/site-packages/torch/optim/_functional.py", line 4, in <module>
    from torch import Tensor
ImportError: cannot import name 'Tensor' from 'torch' (unknown location)
Sender: LSF System <lsfadmin@eu-g3-038>
Subject: Job 217211263: <classification_typicaldec_conservative_50shot> in cluster <euler> Exited

Job <classification_typicaldec_conservative_50shot> was submitted from host <eu-login-17> by user <jmattern> in cluster <euler> at Wed May  4 16:05:54 2022
Job was executed on host(s) <20*eu-g3-038>, in queue <gpu.24h>, as user <jmattern> in cluster <euler> at Wed May  4 16:06:17 2022
</cluster/home/jmattern> was used as the home directory.
</cluster/work/sachan/jmattern/private-datasets> was used as the working directory.
Started at Wed May  4 16:06:17 2022
Terminated at Wed May  4 16:08:13 2022
Results reported at Wed May  4 16:08:13 2022

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
/cluster/work/sachan/jmattern/myenv/bin/python train_classifier.py --gen_data --device 0 --pos-file positive_generated_typical_conservative.txt --neg-file negative_generated_typical_conservative.txt
------------------------------------------------------------

Exited with exit code 1.

Resource usage summary:

    CPU time :                                   6.65 sec.
    Max Memory :                                 303 MB
    Average Memory :                             157.67 MB
    Total Requested Memory :                     20480.00 MB
    Delta Memory :                               20177.00 MB
    Max Swap :                                   -
    Max Processes :                              3
    Max Threads :                                4
    Run time :                                   116 sec.
    Turnaround time :                            139 sec.

The output (if any) follows:

using synth data
['  The strength and majesty of Branagh\'s directing is obvious from just the number of different colors his iconic bears are painted in.  After a fast Chewbacca chase movie, the shooting order gets increasingly out of whack along with the plot: Vader and Chewie descend into the Moscow metro (why\'s no one searching cities in Russia before now?) before Lucas\' Cargo Dog can throw a window at Han\'s back, then Vader barges into the offices of a damned-good comic book publisher while Thór Ó Grádafgís laughs, Han suffers terminal intestinal ulcer, Gueller takes command, and, oops, Han and Chewie fall in love.  The conclusion: Emperor Palpatine orders Vader to do his bidding (doesn\'t any one in this movie exactly what they want?) and the black ice cold Yoda hands Chewie to Luke:  ĭáguk Dagak tranquil gustav, allrayinn bömarvisar í berl ama iarafu, ìgláinta í bolmason ïngär vás." "But make a choice for good, or gain the opposite. The light of the faith warms everlastingly to one who offers an answer to the young Jedi.  Thanks to  THOR banners and colorful bears across the world, MSE7B is going to music LandI from redevelop', '  Right on time with one of my favourite films from last year: BEAUTIFUL people, LADIES, many happy raised voices.... Moral to the picture: Journalists Create the Shit!  Gruff: ', '  I have to give the filmmakers props for finally bringing a woman in a series that\'s been previously all about stuntmen and bad guys in burly boots and tactic swords (see: the terminator series) and having a female part. The actress plays Lucy Simpson perfectly, avoiding any kind of pettishness or histrionics, and giving the best performance going. I adore the "Marty McFly" character (or perhaps Pamela Hewitt, since they both look like dorky lovebirds wearing dresses, but don\'t try to act like you can really talk that way). Another fake Robert Zemeckis short-lived film that\'s worth seeing. I\'m kidding.', '     An excellent romantic comedy, and one that I will definitely see again. No pun intended. Rating:    ', ' _____________________________________________ BECAUSE DEFENSE OF THE FALLEN - The film consist of 4 or 5 episodes: ----------------------- Episode 1: Rardos - Official, Aptitude Recognition Test with Cinematography Episode 2: Piotr Pelcz - Unusual Sight of a Peasant Chapter 1: Lanna! - chess meets the eyes of the King Chapter 2: Confessing an Uncle - A Gangbang! Scene in the pub with OhMyGodFather! _____________________________________________']
[1, 1, 1, 1, 1]
Downloading:   0%|          | 0.00/570 [00:00<?, ?B/s]Downloading: 100%|██████████| 570/570 [00:00<00:00, 530kB/s]
Traceback (most recent call last):
  File "/cluster/work/sachan/jmattern/myenv/lib64/python3.8/site-packages/transformers/configuration_utils.py", line 596, in _get_config_dict
    resolved_config_file = cached_path(
  File "/cluster/work/sachan/jmattern/myenv/lib64/python3.8/site-packages/transformers/utils/hub.py", line 282, in cached_path
    output_path = get_from_cache(
  File "/cluster/work/sachan/jmattern/myenv/lib64/python3.8/site-packages/transformers/utils/hub.py", line 585, in get_from_cache
    http_get(url_to_download, temp_file, proxies=proxies, resume_size=resume_size, headers=headers)
  File "/cluster/apps/nss/gcc-6.3.0/python/3.8.5/x86_64/lib64/python3.8/tempfile.py", line 492, in __exit__
    result = self.file.__exit__(exc, value, tb)
OSError: [Errno 122] Disk quota exceeded

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "train_classifier.py", line 98, in <module>
    model = Model()
  File "train_classifier.py", line 87, in __init__
    self.bert_model = BertModel.from_pretrained('bert-base-uncased')
  File "/cluster/work/sachan/jmattern/myenv/lib64/python3.8/site-packages/transformers/modeling_utils.py", line 1583, in from_pretrained
    config, model_kwargs = cls.config_class.from_pretrained(
  File "/cluster/work/sachan/jmattern/myenv/lib64/python3.8/site-packages/transformers/configuration_utils.py", line 521, in from_pretrained
    config_dict, kwargs = cls.get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "/cluster/work/sachan/jmattern/myenv/lib64/python3.8/site-packages/transformers/configuration_utils.py", line 548, in get_config_dict
    config_dict, kwargs = cls._get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "/cluster/work/sachan/jmattern/myenv/lib64/python3.8/site-packages/transformers/configuration_utils.py", line 636, in _get_config_dict
    raise EnvironmentError(
OSError: Can't load config for 'bert-base-uncased'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure 'bert-base-uncased' is the correct path to a directory containing a config.json file
